config.yaml

```diff
@@ -72,3 +78,7 @@ options:
     type: string
     description: The username if the SMTP server requires authentication.
     default: ''
+  trusted_key_servers:
+    type: string
+    description: Comma separated list of trusted key servers. Defaults to
```

Comment:
Maybe make it clear that it is not the juju configuration option that defaults to `matrix.org`, but the corresponding setting in synapse. Otherwise I would expect a `default` setting here.

Reply:
Sure, thanks